content/docs/user-guide/experiment-management/persisting-experiments.md

```diff
@@ -0,0 +1,99 @@
+# Persisting Experiments
+
+When you run an experiment with `dvc exp run`, it produces temporary objects in
+the Git repository. These are listed with `dvc exp list` and `dvc exp show`.
```

Comment:
> These are listed with `dvc exp list` and `dvc exp show`.

Not about persisting. Should be in another guide I think (which is why I thought https://github.com/iterative/dvc.org/issues/2844 should come first). We can leave it out of this one unless we're using `exp list/show` for examples here.

Reply:
Removed. Thanks.

Reply:
Thank you.